uui-components/src/overlays/Tooltip.tsx

```diff
@@ -1,23 +1,24 @@
 import * as React from 'react';
 import * as reactDom from 'react-dom';
 import cx from 'classnames';
-import * as css from './Tooltip.scss';
-import {Manager, Reference, Popper} from 'react-popper';
-import PopperJS, {Boundary} from "popper.js";
+import * as PropTypes from 'prop-types';
+import { Placement, Boundary } from '@popperjs/core';
+import { Manager, Reference, Popper, PopperChildrenProps } from 'react-popper';
+import type { Options } from '@popperjs/core/lib/modifiers/offset';
 import { uuiElement, IHasCX, LayoutLayer, IHasChildren, UuiContexts, closest } from '@epam/uui';
+import * as css from './Tooltip.scss';
 import { PopperTargetWrapper } from './PopperTargetWrapper';
 import { Portal } from './Portal';
-import * as PropTypes from 'prop-types';
 
 export interface TooltipProps extends IHasCX, IHasChildren {
     content?: any;
     renderContent?(): any;
-    placement?: PopperJS.Placement;
+    placement?: Placement;
     trigger?: 'click' | 'press' | 'hover';
     portalTarget?: HTMLElement;
-    offset?: string;
+    offset?: Options['offset'];
     children?: React.ReactNode;
-    boundaryElement?: Boundary | Element;
+    boundaryElement?: Boundary;
```

Comment:
In Popper 2 'Element' can't be passed?

Reply:
Boundary is typed as HTMLElement, which extends from Element